frontend/src/pages/Login.tsx

```diff
@@ -16,9 +13,6 @@ export default function Login() {
         <Typography variant="h1" gutterBottom>
           PeerPrep
         </Typography>
-        <Button onClick={auth.redirectToSignIn} variant="contained">
-          Log in
-        </Button>
```

Comment:
Are we replacing the login button?

Reply:
We have login button on the navbar, bryan asked if it could be removed